### PR TITLE
Use table values instead of file values for setting up variable in PrePARE

### LINKED ISSUE #367: PrePARE missing  _FillValue exit
Hi,
I have data files with variables which do not include _FillValue and _MissingValue attributes. This causes the following error wihtin PrePARE:
```
Traceback (most recent call last):
...
  File "/work/bm0021/miniconda2/envs/cmor/lib/python2.7/site-packages/cmip6_cv/PrePARE/PrePARE.py", line 446, in ControlVocab
    varmissing = infile.variables[var[0]]._FillValue[0]
AttributeError: _FillValue
```
However, I would like to have an overview about all missing or wrong variable attributes so PrePARE should not exit at this point without checking the other ones. I installed it via conda.
Is there already an option to ignore that error or is it a bug?
I have not looked into the src of PrePARE yet.
Best regards,
Fabi

### PR DESCRIPTION
**Pull request by mauzey1**
Fixes #367

### CODE REVIEW

**Comment by dnadeau4** [comment]
@mauzey1 you can merge it.  Thanks!

**Comment by dnadeau4** [comment]
@mauzey1 you can merge this PR.  Thanks!

**Comment by dnadeau4** [comment]
@durack1 Can you authorized me as well?  Thanks!

File: Src/cmor_CV.c [hunk 1]
```diff
@@ -2466,13 +2467,11 @@ int cmor_CV_variable(int *var_id, char *name, char *units, float *missing,
     }
 
     if (refvar.type == 'i') {
-        int nMissing;
-        nMissing = (int) *missing;
         cmor_vars[vrid].type = 'i';
         cmor_set_variable_attribute_internal(vrid, VARIABLE_ATT_MISSINGVALUES,
-                                         'i', &nMissing);
+                                         'i', &imissing);
```
[the remark this answers is not in the document]
Comment (mauzey1) [reply Src/cmor_CV.c]: This has been fixed.

**Comment by durack1** [comment]
@dnadeau4 this should also work, you're now an admin

**Pull request merged**